src/memray/reporters/templates/__init__.py

```diff
@@ -28,7 +28,9 @@ def include_file(name: str) -> Markup:
     return env
 
 
-def get_report_title(*, kind: str, show_memory_leaks: bool) -> str:
+def get_report_title(*, kind: str, show_memory_leaks: bool, inverted: bool) -> str:
```

Comment:
I think you missed some updates of calls to this function for example here: `tests/unit/test_templates.py`